### PR TITLE
TaskDirectoryActions cleanup; move fetchNodeState to MesosStateActions

### PR DESCRIPTION
**Pull request by mlunoe**
This PR does not change functionality but moves fetchNodeState action to MesosStateActions from TaskDirectoryActions.

### CODE REVIEW

**Comment by mlunoe** [comment]
**mesosphere-ci** Retest this please

File: plugins/services/src/js/events/TaskDirectoryActions.js [hunk 1]
```diff
@@ -4,56 +4,23 @@ import ActionTypes from '../constants/ActionTypes';
...
-    if (node) {
-      pid = node.pid;
-    }
-
-    if (pid) {
-      nodePID = pid.substring(0, pid.indexOf('@'));
-    }
-
-    return `${Config.rootUrl}/agent/${task.slave_id}/${nodePID}/state`;
+  fetchNodeState(task, node, callback) {
```
Comment (orlandohohmeier) [inline plugins/services/src/js/events/TaskDirectoryActions.js]: Can't we completely remove this from the task directory actions?

Comment (mlunoe) [inline plugins/services/src/js/events/TaskDirectoryActions.js]: Actually, you are right. Will update when I have time

File: src/js/events/MesosStateActions.js [hunk 2]
```diff
@@ -37,8 +51,31 @@ var MesosStateActions = {
...
 
+  fetchNodeState: RequestUtil.debounceOnError(
+    Config.getRefreshRate(),
+    function (resolve, reject) {
+      return function (task, node, success, error) {
+        return RequestUtil.json({
+          url: getNodeStateURL(task, node),
+          success(response) {
+            resolve();
+            success(response);
```
Comment (MatApple) [inline src/js/events/MesosStateActions.js]: This is deviating from Flux. Curious why we can't rely on the AppDispatcher to emit the results?

Comment (mlunoe) [inline src/js/events/MesosStateActions.js]: We could. I was just moving code around at this point. It didn't make sense to me that it was listed where it was.

Comment (MatApple) [inline src/js/events/MesosStateActions.js]: So this code was already in our repo?

**Comment by rcorral** [comment]
I've addressed the comments from @MatApple and @orlandohohmeier. Also cleaned up the actions so that we fire different events.
@mlunoe please take a look!

**Comment by rcorral** [comment]
@mlunoe if the changes look good to you, then let's merge!

**Pull request merged**